package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "hypercerts-lexicon",
+  "version": "0.1.0",
+  "description": "ATProto lexicon definitions and TypeScript types for the Hypercerts protocol",
+  "type": "module",
+  "main": "./index.js",
+  "types": "./index.ts",
+  "exports": {
+    ".": {
+      "types": "./index.ts",
+      "import": "./index.ts"
+    }
+  },
+  "files": [
+    "index.ts",
+    "types",
+    "lexicons"
+  ],
+  "keywords": [
+    "hypercerts",
+    "atproto",
+    "lexicon",
+    "typescript"
+  ],
+  "author": "",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "scripts": {
+    "check": "lex gen-api ./types ./lexicons/**/*.json",
```

Comment:
Is this actually doing validation as well as generation?  If so, that's great.  If not, I think it should be called `generate` or `compile` or `build` or similar.

Reply:
Discussed with @s-adamantine and also tested myself - it is indeed validating.